packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -334,8 +334,16 @@ export async function getCommonConfig(wco: WebpackConfigOptions): Promise<Config
     watch: buildOptions.watch,
     watchOptions: {
       poll,
+      // Unlike stated in the docs, this is needed.
+      followSymlinks: buildOptions.preserveSymlinks,
```

Comment:
Since `resolve.symlinks` is disabled when `buildOptions.preserveSymlinks` is enabled, I think the Webpack docs may be correct in suggesting (albeit tersely) that this option (`watchOptions.followSymlinks`) may be needed.

Reply:
You are right, I misread the docs